tests/ValueParsers/MonthNameUnlocalizerTest.php

```diff
@@ -12,11 +12,11 @@
  * @group DataValueExtensions
  * @group ValueParsers
  *
- * @license GPL-2.0+
+ * @license GPL-2.0-or-later
  * @author Addshore
  * @author Thiemo Kreuz
  */
-class MonthNameUnlocalizerTest extends PHPUnit_Framework_TestCase {
+class MonthNameUnlocalizerTest extends PHPUnit\Framework\TestCase {
```

Comment:
The other file does just `extends TestCase`.

Reply:
Thanks, fixed.